cotainr/pack.py

```diff
@@ -292,12 +326,22 @@ def _download_miniforge_installer(self, *, installer_path):
 
         Raises
         ------
+        RuntimeError
+            If the container sandbox architecture is unknown.
         urllib.error.URLError
             If three attempts at downloading the installer all fail.
         """
+        architecture = self.sandbox.architecture
+        if architecture is None:
+            raise RuntimeError(
+                f"Cotainr's CondaInstall got '{architecture=}' "
+                "which indicates that it is not running in a container sandbox context."
+            )
+
+        install_script = CondaInstall._get_install_script(architecture)
```

Comment:
If `_get_install_script` is made a classmethod:
```suggestion
        install_script = self._get_install_script(architecture)
```